book/src/format.md

```diff
@@ -29,6 +29,17 @@ Like built-in derives (e.g. `#[derive(Debug)]`), `#[derive(Format)]` will add `F
 
 > ⚠️ Do *not* use the API used by the expansion of the `derive(Format)` macro; it is *unstable*.
 
+By default the derive assumes `defmt` to exist in your crate's extern prelude.
```

Comment:
I do not understand "your crate's extern prelude". What is that? Can we simplify that a little bit?